clang-tools-extra/test/clang-tidy/checkers/readability/identifier-naming-default-none.cpp

```diff
@@ -0,0 +1,50 @@
+// RUN: %check_clang_tidy %s readability-identifier-naming %t -- \
+// RUN:   -config='{CheckOptions: {}}' \
+// RUN:   -- -fno-delayed-template-parsing
+
+// Empty options effectively disable the check, allowing PascalCase...
+#define MyMacro
+
+namespace MyNamespace {
+
+using MyAlias = int;
+
+int MyGlobal;
+
+struct MyStruct {
+  int MyField;
+};
+
+template <typename MyTypename>
+int MyFunction(int MyArgument) {
+  int MyVariable = MyArgument;
+  return MyVariable;
+}
+
+template int MyFunction<int>(int);
+
+}
```

Comment:
Will be good idea to add closing comment. Same for other namespaces in this and another test.